### DISCUSSION TITLE
Access an old generated payload link ?

### DISCUSSION THREAD

**Discussion by Sim4n6** [projectdiscovery/interactsh#110] [Q&A] [answered]
I hosted your interactsh-server on a a droplet. Everything is working like a charm. But I have a question: 
Can I use an old generated payload link ? access its http/dns/smtp requests within the server using the client side (interactsh-client ? please ?

**Comment by ehsandeep** [comment, marked as answer]
@Sim4n6 Yes you can, as long as the client is running, the payload should be working but it depends on what client you are using, for example, CLI clients deregister old payloads as soon as you close/cancel the CLI client, [Web Client](https://github.com/projectdiscovery/interactsh-web) keeps the payload active for always, so it's more practical to use web client for this use case.
